nodewatcher/modules/analysis/rogue_nodes/tests.py

```diff
@@ -0,0 +1,28 @@
+import json
+import io
+import os
+
+from django import test as django_test
+
+from nodewatcher.modules.analysis.rogue_nodes.tasks import rogue_node_detection_algorithm
+
+
+class RogueDetectionTest(django_test.TestCase):
+
+    def test_rogue_detection_algorithm(self):
+        print('testing rogue_node_detection_algorithm')
```

Comment:
After much Googling:
By "dynamically creating a test suite": Do you mean a function that takes in a specific input and asserted results, then processes the specific input and compares it to asserted results? 

(something like "test_animal_pk" on https://docs.djangoproject.com/en/1.9/topics/testing/advanced/)

And then call this function with all the test cases? So have 8 function calls like test(e2-study-room), test(living-room), ... ?